src/render/copy2clipboard.js

```diff
@@ -0,0 +1,35 @@
+function oldCopyTextToClipboard(text) {
```

Comment:
Maybe it would be good to rename this function, e.g. to `legacyCopyTextToClipboard`?

With the current name, I was tempted to check whether the function is actually used, or whether it is a remnant only and can be removed.

Reply:
Renamed as `defaulCopyTextToClipboard`